packages/replay/src/util/addEvent.ts

```diff
@@ -81,6 +88,33 @@ export async function addEvent(
   }
 }
 
+function shouldAddEvent(replay: ReplayContainer, event: RecordingEvent): boolean {
```

Comment:
Might be good to have a unit test for this

Reply:
I've added tests for this!